doc/REQUIRE

```diff
@@ -1,3 +1,3 @@
 Compat 0.25.0 0.25.0+
```

Comment:
let's also bump this to `0.25.2 0.25.2+` to get rid of some deprecation warnings